tests/data/western_usa_live_fuel_moisture/data.py

```diff
@@ -172,7 +172,7 @@
             'type': 'text/csv',
         },
     },
-    'bbox': [-115.8855556, 42.44111111, -115.8855556, 42.44111111],
+    'bounds': [-115.8855556, 42.44111111, -115.8855556, 42.44111111],
```

Comment:
I'm guessing this was changed by accident?

Reply:
It was